AzureMonitorAgent/agent.py

```diff
@@ -320,7 +320,15 @@ def copy_mdsd_fluentbit_binaries():
     fluentbit_bin = "/opt/microsoft/azuremonitoragent/bin/fluent-bit"
 
     # copy the required libs to our test directory first
-    copytree("/opt/microsoft/azuremonitoragent/lib", os.getcwd() + "/lib")
+    lib_dir = os.path.join(os.getcwd(), "lib")
+    if os.path.exists(lib_dir):
+        rmtree(lib_dir)
```

Comment:
is there anything we should do to check `lib_dir` is the path we intend? this may be overthinking, but i'm imagining accidentally removing `/lib`